src/js/ui/otb/actions.js

```diff
@@ -7,6 +7,7 @@ import ground from '../shared/offlineRound/ground';
 import popupWidget from '../shared/popup';
```

Comment:
haha :) 

When I'm done with chessground (already rewritten by @ornicar ) and puzzles we'll have almost all lichobile in typescript. That will be much better, since as long you keep some javascript you always risk to break something